src/intersections/rays.jl

```diff
@@ -97,21 +99,22 @@ end
 
 # Williams A, Barrus S, Morley R K, et al., 2005.
 # (https://dl.acm.org/doi/abs/10.1145/1198555.1198748)
-function intersection(f, ray::Ray{Dim,T}, box::Box{Dim,T}) where {Dim,T}
-  invdir = one(T) ./ (ray(1) - ray(0))
+function intersection(f, ray::Ray{Dim}, box::Box{Dim}) where {Dim}
+  invdir = 1 ./ (ray(1) - ray(0))
```

Comment:
we need to be consistent regarding 0 and 1 in the code. I believe using `zero`, `one` and `oneunit` always is the easiest rule to remember.